inst/@sym/horzcat.m

```diff
@@ -62,7 +62,7 @@
           'return sp.Matrix.hstack(*_proc),'
           };
 
-  varargin = sym(varargin);
+  varargin = cell_array_to_sym(varargin);
```

Comment:
Instead of reopening octave you can do "clear classes". It will do clear all and then reload classes. 'rehash' would probably work too if you don't want to clear the workspace.